VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ClipboardService } from 'ngx-clipboard';
+import { VhoQueryService } from '../services/vho-query-service.service';
+
+@Component({
+    selector: 'app-copy-quick-link',
+    templateUrl: './copy-quick-link.component.html',
+    styleUrls: ['./copy-quick-link.component.scss']
+})
+export class CopyQuickLinkComponent implements OnInit {
+    @Input() conferenceId: string;
+    hearingId: string;
+    tooltip: string;
+    tooltipCopiedText = 'Join by quick link details copied to clipboard';
+    tooltipText = 'Copy join by quick link details to clipboard';
```

Comment:
These should be in i18n files.

Reply:
These two strings where left and after speaking with Muki these can now be removed.